src/box/iproto.cc

```diff
@@ -2624,7 +2684,9 @@ tx_process_replication(struct cmsg *m)
 		  * written row. Do not push it on top.
 		  */
 	} catch (Exception *e) {
-		iproto_write_error(io, e, ::schema_version, msg->header.sync);
+		if (!iproto_shutdown_started)
+			iproto_write_error(io, e, ::schema_version,
```

Comment:
Better catch the proper exception type here and skip logging for it, like we do for socket and timeout errors.

Reply:
Changed to:

```diff
@@ -2621,6 +2701,8 @@ tx_process_replication(struct cmsg *m)
                  * In case of a timeout the error could come after a partially
                  * written row. Do not push it on top.
                  */
+       } catch (FiberIsCancelled *e) {
+               /* Do not write into connection on connection drop. */
        } catch (Exception *e) {
                iproto_write_error(io, e, ::schema_version, msg->header.sync);
        }
```